Fix documentation after change of default locality in #16258. (#17385)
coq/coq: doc/sphinx/addendum/type-classes.rst

   This command supports the :attr:`local`, :attr:`global` and :attr:`export`
   locality attributes.

   .. versionchanged:: 8.18

>       The default value for instance locality outside sections used to be :attr:`global`.
>       This was changed to :attr:`export`.

Reviewer: ```suggestion
      The default value for instance locality outside sections is now :attr:`export`.
      It used to be :attr:`global`.
```

The history is less important that the first sentence, so swap.
